### ISSUE TITLE
Collaborator needed/Collaborators exchange

### ISSUE THREAD

**Issue by ows-ali** [ows-ali/Hacktoberfest#24]
I am looking for a collaborator for this repository with following conditions:
- Never a collaborator before
- Make me collaborator of their owned repository to expand Github knowledge

**Comment by 73VW** [comment]
I can be your collaborator. My Hacktoberfest participating repository is the following: https://github.com/73VW/Publishing-to-PyPI-with-pbr-and-Travis
Tell me if you're interrested.
Cheers

**Comment by 73VW** [comment]
I confirm that 73VW is my username.

But just a question first.

By this : 

> Never a collaborator before

You mean never before on github or never before on this repository?

Because I have already experience as collaborator on github and I will let this role to someone else with pleasure.

**Comment by ows-ali** [comment]
Yes, this is mandatory condition as mentioned. I am looking for someone who was never a collaborator on Github before so that it will be new experience for both of us.
Thanks for your interest by the way 🌝

**Comment by 73VW** [comment]
I am sorry, I didn't understand correctly. Have a nice day :)

**Comment by ows-ali** [comment]
You too 👍

**Comment by ows-ali** [comment]
@m-sameer why not, do you have any repository that you can make me collaborator of?

**Comment by ows-ali** [comment]
Have you been a collaborator before? it is important that you do not have collaborator experience

**Comment by m-sameer** [comment]
I am a collaborator of [this repo](https://github.com/BaReinhard/Hacktoberfest-Data-Structure-and-Algorithms) but I don't do much as it's so huge.
And I don't have any repo which is mine for Hacktober.

**Comment by ows-ali** [comment]
Oh so it is going against  both conditions mentioned. 
But we can work this out if you are ready to follow this strategy. 
You can make a new repository welcoming contributions for Hacktoberfest and make me collaborator. Come up with an idea and write a README file only.

**Comment by m-sameer** [comment]
You know, just leave it. I'll be busy this quarter. Then, the repo I would create would be there, rotting, for years. 
Thanks.

**Comment by ows-ali** [comment]
Bro I am also trying this first time. You can check my profile. @m-sameer

**Comment by ows-ali** [comment]
@codingCookie This is also open :)

**Comment by ows-ali** [comment]
@seancyw

**Comment by seancyw** [comment]
What does collaborator need to do? @ows-ali

**Comment by ows-ali** [comment]
Just merge the PRs if they are according to the requirements or solving any issue and some other basic management. You can become the collaborator if you have not served as collaborator before

**Comment by matias9477** [comment]
Can I help you with anything?

**Comment by cryptographicfool** [comment]
Are you still looking for a collaborator?

**Comment by ahmad380360** [comment]
اشكركم علا الهتمام. دمج قريب

**Comment by sushangagneja** [comment]
Interested!

**Comment by Blahblahblahsomething** [comment]
Is this still open?

**Comment by ows-ali** [comment]
Hello all, whoever is interested in becoming the collaborator of this project, please email me your Github username at owaisali.cs@gmail.com and please make sure you know about Hacktoberfest, Git and rules of this repo. Please tell me shortly how will you merge PRs, resolve conflicts etc in email. Thanks

**Comment by GouravRusiya30** [comment]
I am interested to become collaborator @ows-ali 
I have sent you an email, please check n let me know.

**Comment by Scienticious** [comment]
@ows-ali I am also interested

**Comment by ows-ali** [comment]
@Scienticious email at owaisali.cs@gmail.com

**Comment by Scienticious** [comment]
@ows-ali have already sent you the email

**Comment by somayaj** [comment]
Willing to contribute, is this still open?

**Comment by namankhurpia** [comment]
I m willing to contribute to hacktoberfest, please please please accept my PR

**Comment by bunish** [comment]
Hey! I want to contribute to this project. Please assign it to me.

**Comment by RajasBharambe** [comment]
Hey ! Can I contribute to this project ??
Please assign it to me

**Comment by sykalerio** [comment]
Hi there,

I'm Elpida  a new contributor interested in helping out with this issue. I'm excited to learn more about testing and would love to get some guidance on how to proceed.

Could you please provide me with more information on what specifically needs to be tested in this issue? Also, if you have any resources or tutorials on testing with Jest that would be really helpful.

Thanks, and looking forward to contributing!

Best,
Elpida

**Comment by kedhar0608** [comment]
Is this still available I want to join. I am new.

**Comment by vimalJD** [comment]
### Is this project is ongoing? Let me know further (DOCS) to contribute on it as "good first issue".

**Comment by avnish-kt** [comment]
I am also new to open source contributions I just did a pull request on #1 to start with my contributions. I am very excited to join and make some progress in open source contributions.

Thanks,
Avnish

**Comment by Dxuian** [comment]
still open ?

**Comment by S-YED** [comment]
i wanna be collaborator as well as Contributor @ows-ali are you available ?
